selfdrive/ui/qt/onroad.cc

```diff
@@ -163,10 +169,13 @@ void ButtonsWindow::updateState(const UIState &s) {
     dfStatus = s.scene.dfButtonStatus;
     dfButton->setStyleSheet(QString("font-size: 45px; border-radius: 100px; border-color: %1").arg(dfButtonColors.at(dfStatus)));
 
-    MessageBuilder msg;
-    auto dfButtonStatus = msg.initEvent().initDynamicFollowButton();
-    dfButtonStatus.setStatus(dfStatus);
-    QUIState::ui_state.pm->send("dynamicFollowButton", msg);
+    std::string toyota_distance_btn = util::read_file("/data/community/params/toyota_distance_btn");
```

Comment:
Since this is called once every update frame can we get the param once on init and then just use that? You can try `dfButton.visible` or just a class boolean